### PR TITLE
feat: support deletion of Cloud Asset Inventory feeds not in use in organization

### PR DESCRIPTION
**Pull request by renato-rudnicki**
This PR supports the deletion of Cloud Asset Inventory feeds not in use in organization that match a given pattern

- Added fix for [issue 183](https://github.com/terraform-google-modules/terraform-google-scheduled-function/issues/183)

### CODE REVIEW

File: modules/project_cleanup/function_source/main.go [hunk 1]
```diff
@@ -175,6 +181,19 @@ func checkIfTagKeyShortNameExcluded(shortName string, excludedTagKeys []string)
 	return false
 }
 
+func checkIfCaiFeedsShortNameIncluded(shortName string, includedFeeds []*regexp.Regexp) bool {
```
Comment (bharathkkb) [inline modules/project_cleanup/function_source/main.go:184]: Looks like @daniel-cit also has a similar func in https://github.com/terraform-google-modules/terraform-google-scheduled-function/pull/196/. Could we use a common checkIfNameIncluded(name string, reg []*regexp.Regexp) func?

File: modules/project_cleanup/function_source/main.go [hunk 2]
```diff
@@ -224,6 +243,46 @@ func getCleanUpTagKeysOrTerminateExecution() bool {
...
+		logger.Printf("Failed to get CAI Feeds list from [%s] env variable, error [%s]", envVariableName, err.Error())
+		return nil
+	}
+
+	regexFeeds := make([]*regexp.Regexp, len(caiFeeds))
+	for i, feed := range caiFeeds {
+		regexFeeds[i], err = regexp.Compile(feed)
+		if err != nil {
+			logger.Printf("Failed to compile regex for CAI Feed [%s], error [%s]", feed, err.Error())
+			regexFeeds[i] = regexp.MustCompile(".*")
```
Comment (bharathkkb) [inline modules/project_cleanup/function_source/main.go:266]: Could we skip adding to `regexFeeds[i]` if err?

**Review by bharathkkb — APPROVED** [review]
LGTM minor comment about rebasing on Daniels PR to share some of the code

File: modules/project_cleanup/function_source/main.go (hunk 2 above)
Comment (renato-rudnicki) [reply modules/project_cleanup/function_source/main.go]: fixed.

File: modules/project_cleanup/function_source/main.go (hunk 1 above)
Comment (renato-rudnicki) [reply modules/project_cleanup/function_source/main.go]: done.

**Comment by daniel-cit** [comment]
@bharathkkb @apeabody Build is Green 🟢

**Pull request merged**